conformance/Makefile

```diff
@@ -32,22 +33,18 @@ create-kind-cluster: ## Create a kind cluster
 	kind create cluster --image $(KIND_IMAGE)
 	kind export kubeconfig --kubeconfig $(KIND_KUBE_CONFIG)
 
-.PHONY: preload-nginx-container
-preload-nginx-container: ## Preload NGINX container on configured kind cluster
-	docker pull $(NGINX_IMAGE)
-	kind load docker-image $(NGINX_IMAGE)
-
 .PHONY: update-nkg-manifest
-update-nkg-manifest: ## Update the NKG deployment manifest image name and imagePullPolicy
-	cd .. && make generate-manifests HELM_TEMPLATE_EXTRA_ARGS_FOR_ALL_MANIFESTS_FILE="--set nginxGateway.kind=skip" HELM_TEMPLATE_COMMON_ARGS="--set nginxGateway.image.repository=$(NKG_PREFIX) --set nginxGateway.image.tag=$(NKG_TAG) --set nginxGateway.image.pullPolicy=Never" && cd -
+update-nkg-manifest: ## Update the NKG deployment manifest image names and imagePullPolicies
+	cd .. && make generate-manifests HELM_TEMPLATE_EXTRA_ARGS_FOR_ALL_MANIFESTS_FILE="--set nginxGateway.kind=skip" HELM_TEMPLATE_COMMON_ARGS="--set nginxGateway.image.repository=$(NKG_PREFIX) --set nginxGateway.image.tag=$(NKG_TAG) --set nginxGateway.image.pullPolicy=Never --set nginx.image.repository=$(NGINX_IMAGE_NAME) --set nginx.image.tag=$(NKG_TAG) --set nginx.image.pullPolicy=Never" && cd -
 
 .PHONY: build-nkg-image
-build-nkg-image: ## Build NKG container and load it and NGINX container on configured kind cluster
-	cd .. && make PREFIX=$(NKG_PREFIX) TAG=$(NKG_TAG) container
+build-nkg-images: ## Build NKG and nginx images and load them on the kind cluster
```

Comment:
because README includes the generated help, we need to update it too to include the new target